.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+
+**What type of PR is this?**
+> Uncomment only one and also add the corresponding label in the PR:
+>bug
+>feature
+>cleanup
+>api-change
+>design
+>documentation
+>failing-test
+>unit-test
+>integration-test
+
+**What this PR does / why we need it**:
+
+
+**Does this PR change a user-facing CRD or CLI change?**:
+<!--
+If yes, explain why the change is needed and paste some example output of the new change.
+If no, just write no.
+-->
+
+**Is a release note needed?**:
+<!--
+If yes, add the release-note label to the PR. Also enter a single sentence release-note block below.
+If no, just write no and remove the release-note block below.
+-->
+```release-note
+
+```
+
+**Does the PR change need to be cherry-picked to a release branch?**:
+<!--
+If yes, enter a comma-separated list of branches to cherry pick.
```

Comment:
`If yes, enter a comma-separated list of branches where it should be cherry-picked`